charts/caluma/templates/ingress.yaml

```diff
@@ -2,9 +2,9 @@
 {{- $fullName := include "caluma.fullname" . -}}
 {{- $svcPort := .Values.service.port -}}
 {{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
Nowadays Kubernetes releases which don't support `networking.k8s.io/v1` should be very rare, so I would suggest to remove the whole if/else section.